### ISSUE TITLE
Add support for changing the casing of the decorator for use in the fixer

### ISSUE THREAD

**Issue by ChiriVulpes** [hmil/tslint-override#14]
The rest of the decorators we use in my project are capitalised. It's nice that this tslint rule still allows those to pass, but it'd be nicer if the fixer had a way to work with that style as well. 

Maybe support passing a string option that must start with `@`, and that is the name of the decorator? IE:
```json
"explicit-override": [true, "decorator", "exclude-interfaces", "@Override"],
```

But I guess then someone could change it to something meaningless like `@Foo`, maybe you wouldn't want to do it that way for that reason

**Comment by ChiriVulpes** [comment]
Implemented in https://github.com/Yuudaari/tslint-override

I can make a PR later if you like, if this way about going about it looks ok to you (I assume I should add tests too)

**Comment by hmil** [comment]
Sure, that sounds like a reasonable feature. Feel free to open a PR with the work you've done.

**Comment by hmil** [comment]
I did a bunch of changes to fix your other issue #15 . You might want to rebase your change on top of master.
